.travis/run_tests.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -euo pipefail
+
+: ${1?"Usage: $0 TEST_NAME (one of {ASAN, DEBUG})"}
+TEST_NAME=${1}
+
+# Note: keep_alive() prevents travis from exiting early on tests that take a while.
+keep_alive() { while true; do echo -en "\a"; sleep 540; done }
+
+# Pass in which tests to run (one of {test, test_unopt})
+run_unit_tests() {
+    CTEST_PARALLEL_LEVEL=2 ninja ${1} || ( cat Testing/Temporary/LastTest.log && exit 1 )
+}
+
+# Pass in YES/NO for QUANTIZE
+run_and_check_bundle() {
+    echo "Checking lenet_mnist bundle with QUANTIZE=${1}"
+    cd ${GLOW_SRC}/examples/bundles/lenet_mnist/
+    ( QUANTIZE=${1} make &> raw_results.txt ) || ( cat raw_results.txt && exit 1 )
+    ( tail -n72 raw_results.txt | grep -F "Result: " > results.txt ) || ( cat raw_results.txt && exit 1 )
+    diff results.txt ${GLOW_SRC}/.travis/lenet_mnist_expected_output.txt
```

Comment:
if diff returns something then we'll return error code back to Travis, right? (no need to explicitly exit 1)

Reply:
Yeah that was the intention, I believe it should work like that. I'll verify. I suppose I could also explicitly `exit 1` but it should work correctly like that.

Reply:
Yep, looks like it works as expected: https://travis-ci.org/pytorch/glow/jobs/442479011